app/main.py

```diff
@@ -1,8 +1,64 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one, the closing bracket is also written on a new line

Reply:
1 and 2 done